### DISCUSSION TITLE
Exception rule for home page URL (default site/language)

### DISCUSSION THREAD

**Discussion by lightbringer93** [statamic/cms#5955] [Q&A]
In Switzerland we often have websites with multiple languages. 

For example, we recently launched ao-kreston.ch with 3 languages ( DE / EN / FR ). 
In `sites` we defined /de /en /fr. But now the start page (Home) of the default language is "/de".
Is there any way to change this page to just "/", but without removing the prefix "/de" from the subpages?

**Comment by edalzell** [comment]
Let's see your `sites.php` please.

**Comment by lightbringer93** [comment]
Sure!
```
<?php

return [

    /*
    |--------------------------------------------------------------------------
    | Sites
    |--------------------------------------------------------------------------
    |
    | Each site should have root URL that is either relative or absolute. Sites
    | are typically used for localization (eg. English/French) but may also
    | be used for related content (eg. different franchise locations).
    |
    */

    'sites' => [

        'de' => [
            'name' => 'Deutsch',
            'locale' => 'de_CH',
            'url' => env('APP_URL') . 'de/',
        ],

        'en' => [
            'name' => 'Englisch',
            'locale' => 'en_US',
            'url' => env('APP_URL') . 'en/',
        ],

        'fr' => [
            'name' => 'Französisch',
            'locale' => 'fr_FR',
            'url' => env('APP_URL') . 'fr/',
        ],

    ],
];
```

**Comment by joshuablum** [comment]
Hi Demien,

If I understand you correctly the URL `ao-kreston.ch` should essentially show `ao-kreston.ch/de`, right? You just want a prettier URL for the home / root page. All other URLs / deeper nested sites should show the `/de` path, correct?
From what I know that's not possible with Statamic alone (don't even think Laravel can do that).
One solution might be to do it on the server level and configure a nginx alias? More on that [here](https://nginx.org/en/docs/http/ngx_http_core_module.html#alias).

Could look similar to this:
```
location / {
    alias /de/;
}
```

There is potential that it can break something though, so be cautious.

If you use Laravel Forge you can edit the file from the web interface or most of the time the file is located at `/etc/nginx/nginx.conf`. Don't forget to reload it after making changes.

**Reply by lightbringer93** [reply]
Thank you for the reply and yes you understand correctly the URL should only be diffrent for the home / root page. 

We solved the problem with a 301 redirect, but we were wondering if there was a diffrent solution. Since this is quite common for our projects.

**Reply by edalzell** [reply]
put the "default" site at `/` with an nginx rule to forward `/de` to `/`

**Reply by lightbringer93** [reply]
Thank you but we don't want that since every site on DE would than be a redirect, which would be quite bad for SEO.
